From: Philipp Zabel <p.zabel@pengutronix.de>
Date: Thu, 7 Jan 2016 16:41:23 +0100
Subject: [PATCH] drm: bridge/dw_hdmi: detect initial connector state

Allow userspace to read the initial connector space via sysfs without having
to issue a detect.

Signed-off-by: Philipp Zabel <p.zabel@pengutronix.de>
---
 drivers/gpu/drm/bridge/synopsys/dw-hdmi.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/gpu/drm/bridge/synopsys/dw-hdmi.c b/drivers/gpu/drm/bridge/synopsys/dw-hdmi.c
index cceb5aab6c83..1c0cc8903455 100644
--- a/drivers/gpu/drm/bridge/synopsys/dw-hdmi.c
+++ b/drivers/gpu/drm/bridge/synopsys/dw-hdmi.c
@@ -2599,6 +2599,7 @@ static int dw_hdmi_connector_create(struct dw_hdmi *hdmi)
 	if (hdmi->version >= 0x200a && hdmi->plat_data->use_drm_infoframe)
 		drm_connector_attach_hdr_output_metadata_property(connector);
 
+	hdmi->connector.status = dw_hdmi_connector_detect(connector, false);
 	drm_connector_attach_encoder(connector, hdmi->bridge.encoder);
 
 	cec_fill_conn_info_from_drm(&conn_info, connector);
